libs/parallelism/algorithms/tests/unit/algorithms/unique_copy_tests.hpp

```diff
@@ -118,6 +118,29 @@ struct random_fill
 };
 
 ///////////////////////////////////////////////////////////////////////////////
+template <typename IteratorTag, typename DataType, typename Pred>
+void test_unique_copy(IteratorTag, DataType, Pred pred, int rand_base)
+{
+    typedef typename std::vector<DataType>::iterator base_iterator;
```

Comment:
We try to use `using =` instead of `typedef`, there are still some `typedef`s in the code but we replace them when modifying the code nearby; it's not a must for this PR but it's good to keep it in mind ;)

Reply:
Oh ok I will, thanks!